app/controllers/backend/tax_declarations_controller.rb

```diff
@@ -64,7 +64,12 @@ def index
 
       respond_to do |format|
         format.html do
-          render "alert_no_VAT_declaration" unless FinancialYear.where.not(tax_declaration_mode: "none").any?
+          not_opened_financial_year = FinancialYear.opened.empty?
+          opened_financial_year_with_tax_declaration_mode_none = !FinancialYear.with_tax_declaration.include?(FinancialYear.current)
+          vat_declarations_fullfilled = !FinancialYear.current.missing_tax_declaration? if FinancialYear.current
```

Comment:
If you don't have a `FinancialYear.current` this will make the `vat_declarations_fullfilled` variable `nil` (which is falsey) is this the intended behaviour?
If it is, I'd rather we make it explicit: `FinancialYear.current && !FinancialYear.current.missing_tax_declaration?`